content/Mobile/Android/index.adoc

```diff
@@ -15,7 +15,7 @@ Yubico Android SDK (YubiKit for Android) is an Android library provided by Yubic
 Documentation for the SDK, such as instructions on adding it to your project and getting started, is available
 link:https://github.com/Yubico/yubikit-android[on GitHub].
 
-JavaDoc API's for the various modules can be found link:https://developers.yubico.com/yubikit-android/JavaDoc/[here].
+JavaDoc APIs for the various modules can be found link:../../yubikit-android/JavaDoc/[here].
```

Comment:
Would linking to "/yubikit-android/JavaDoc/" be more robust?

Reply:
I will test this, thanks for the suggestion.

Reply:
worked great! Fixed